src/AWS.Deploy.ServerMode.Client/RestAPI.cs

```diff
@@ -2190,21 +2190,6 @@ public partial class StartDeploymentSessionOutput
     [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.4.1.0 (Newtonsoft.Json v13.0.0.0)")]
     public partial class SystemCapabilitySummary 
     {
-        [Newtonsoft.Json.JsonProperty("name", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
-        public string Name { get; set; }
```

Comment:
This might be due to the removal of "{ get; set; }"
Please ensure the Rest API is emitting the expected properties, and add an integration test to help guard against this kind of accident. This would break compilation if a release was integrated into a client.

Reply:
I could call the `GetCompatibility` API in one of our existing integrations tests but the result would always be an empty list as all the dependencies are satisfied.

I looked into modifying the environment variables to simulate the absence of Node.js or Docker. However, our CI system runs on a Linux machine and .NET implementation on UNIX based OS only supports manipulations on process-scoped environment variables.

For now, to ensure the API response is as expected, I performed a manual test by deleteing Node.js from my system

```
public async Task CheckCompatibilities()
        {
            var projectPath = _testAppManager.GetProjectPath(Path.Combine("testapps", "WebAppWithDockerFile", "WebAppWithDockerFile.csproj"));
            var portNumber = 4012;
            using var httpClient = ServerModeHttpClientFactory.ConstructHttpClient(ResolveCredentials);

            var serverCommand = new ServerModeCommand(_serviceProvider.GetRequiredService<IToolInteractiveService>(), portNumber, null, true);
            var cancelSource = new CancellationTokenSource();

            var serverTask = serverCommand.ExecuteAsync(cancelSource.Token);
            try
            {
                var baseUrl = $"http://localhost:{portNumber}/";
                var restClient = new RestAPIClient(baseUrl, httpClient);

                await WaitTillServerModeReady(restClient);

                var startSessionOutput = await restClient.StartDeploymentSessionAsync(new StartDeploymentSessionInput
                {
                    AwsRegion = _awsRegion,
                    ProjectPath = projectPath
                });

                var sessionId = startSessionOutput.SessionId;
                var getRecommendationOutput = await restClient.GetRecommendationsAsync(sessionId);
                var fargateRecommendation = getRecommendationOutput.Recommendations.FirstOrDefault(x => string.Equals(x.RecipeId, "AspNetAppEcsFargate"));

                await restClient.SetDeploymentTargetAsync(sessionId, new SetDeploymentTargetInput
                {
                    NewDeploymentRecipeId = fargateRecommendation.RecipeId,
                    NewDeploymentName = "MyAppStack"
                });

                var result = await restClient.GetCompatibilityAsync(sessionId);

                Assert.Equal(1, result.Capabilities.Count);

                var capability = result.Capabilities.First();

                Assert.Equal("Node.js", capability.Name);
                Assert.Equal("https://nodejs.org/en/download/", capability.InstallationUrl);
                Assert.False(string.IsNullOrEmpty(capability.Message));
            }
            finally
            {
                cancelSource.Cancel();
            }
        }
```